privacyidea/lib/eventhandler/responsemangler.py

```diff
@@ -0,0 +1,164 @@
+                             "description": _("The type of the value."),
+                             "value": ["string", "integer", "bool"]
+                            },
+                        "value":
+                            {"type": "str",
+                             "required": True,
+                             "description": _("The value of the JSON key that should be added.")
+                            }
+                        }
+                   }
+        return actions
+
+    def do(self, action, options=None):
+        """
+        This method executes the defined action in the given event.
+
+        :param action:
+        :param options: Contains the flask parameters g, request, response
+            and the handler_def configuration
+        :type options: dict
+        :return:
+        """
+        ret = True
+        response = options.get("response")
+        content = self._get_response_content(response)
+        handler_def = options.get("handler_def")
+        handler_options = handler_def.get("options", {})
+
+        json_pointer = handler_options.get("JSON pointer", "")
+        value = handler_options.get("value")
+        type = handler_options.get("type")
+        comp = json_pointer.split("/")
+        comp = [x for x in comp if x]
+        if action.lower() == ACTION_TYPE.DELETE:
+            try:
+                if len(comp) == 1:
+                    del(content[comp[0]])
+                elif len(comp) == 2:
+                    del(content[comp[0]][comp[1]])
+                elif len(comp) == 3:
+                    del (content[comp[0]][comp[1]][comp[2]])
+                else:
+                    log.warning("JSON pointer length of {0!s} not supported.".format(len(comp)))
+                options.get("response").data = json.dumps(content)
+            except KeyError:
+                log.warning("Can not delete response JSON Pointer {0!s}.".format(json_pointer))
+        elif action.lower() == ACTION_TYPE.ADD:
+            try:
+                if type == "integer":
+                    value = int(value)
+                elif type == "bool":
+                    value = is_true(value)
+            except ValueError:
+                log.warning("Failed to convert value")
+            try:
+                if len(comp) == 1:
+                    content[comp[0]] = value
+                elif len(comp) == 2:
+                    content[comp[0]] = content.get(comp[0]) or {}
+                    content[comp[0]][comp[1]] = value
+                elif len(comp) == 3:
+                    content[comp[0]] = content.get(comp[0]) or {}
+                    content[comp[0]][comp[1]] = content[comp[0]].get(comp[1]) or {}
+                    content[comp[0]][comp[1]][comp[2]] = value
+                else:
+                    log.warning("JSON pointer of length {0!s} not supported.".format(len(comp)))
+                options.get("response").data = json.dumps(content)
```

Comment:
Now this gets a little tricky: In case we have an endpoint that does not return JSON data (`/validate/radiuscheck`, `/token` with `outform=csv`) we would overwrite possible data here.
Maybe we should only trigger the mangler if the content-type is `application/json`?

Reply:
The ``self._get_response_content(response)`` actually allows parsing either real json response or text/html as json. 
So I decided to catch the exception in case the response is not jsonable.
Done in 0e16f5f5

Reply:
I've drafted PR #1851 to return json by default. This would also change the behavior of `_get_response_content()`. But we can discuss this later.